Code/JavaWrappers/gmwrapper/src-test/org/RDKit/FMCSTests.java

```diff
@@ -1,20 +1,20 @@
-/* 
+/*
```

Comment:
ah my editor's whitespace removal has made a lot changes here, I'll fix that once tests pass